internal/providers/gitlab/selector_entity.go

```diff
@@ -1,44 +0,0 @@
-//
-// Copyright 2024 Stacklok, Inc.
```

Comment:
In a follow-up, I'm going to remove the provider-specific interface and just move the repo,PR and artifact conversion to provider-generic code because with the addition of properties, we don't really need anything provider-specific.